test/commands/workflow/run.test.ts

```diff
@@ -91,6 +96,56 @@ describe("commands/workflow/run", () => {
                 "api-origin": undefined,
                 environment: "staging",
                 recipients: ["alice", "barry"],
+                "run-local": false,
+              }),
+          ),
+        );
+      });
+  });
+
+  const currCwd = process.cwd();
+  const workflowJsonFile = "new-comment/workflow.json";
+
+  describe("given the workflow key arg and the run-local flag", () => {
+    beforeEach(() => {
+      fs.removeSync(sandboxDir);
+      fs.ensureDirSync(sandboxDir);
+      const abspath = path.resolve(sandboxDir, workflowJsonFile);
+      fs.outputJsonSync(abspath, { name: "New comment" });
+
+      process.chdir(sandboxDir);
+    });
+
+    afterEach(() => {
+      process.chdir(currCwd);
+      fs.removeSync(sandboxDir);
+    });
+
+    setupWithStub({ data: { workflow: factory.workflow() } })
+      .stdout()
+      .command([
+        "workflow run",
+        "new-comment",
+        "--environment",
+        "staging",
+        "--recipient",
+        "alice,barry",
+        "--run-local",
+      ])
+      .it("calls apiV1 runWorkflow with expected props", () => {
+        sinon.assert.calledWith(
+          KnockApiV1.prototype.runWorkflow as any,
+          sinon.match(
+            ({ args, flags }) =>
+              isEqual(args, {
+                workflowKey: "new-comment",
+              }) &&
+              isEqual(flags, {
+                "service-token": "valid-token",
+                "api-origin": undefined,
+                environment: "staging",
+                recipients: ["alice", "barry"],
+                "run-local": true,
```

Comment:
We should test the second arg (i.e. workflow input) being passed into the `runWorkflow` function.